lib/python/rose/suite_engine_procs/cylc.py

```diff
@@ -1021,10 +1021,10 @@ def get_suite_state_summary(self, user_name, suite_name):
         dao = self._db_init(self.SUITE_DB, user_name, suite_name)
         if not os.access(dao.db_f_name, os.F_OK | os.R_OK):
             return ret
-        stmt = "SELECT time FROM task_events ORDER BY time DESC LIMIT 1"
-        for row in self._db_exec(self.SUITE_DB, user_name, suite_name, stmt):
-            ret["last_activity_time"] = row[0]
-            break
+
+        ret["last_activity_time"] = strftime("%Y-%m-%dT%H:%M:%S+0000",
```

Comment:
I can do. There's no requirement either way, that's just the way its already done on the existing suite listing page (and here), so I can change both lots accordingly. @matthewrmshin - are you happy for me to change that?